config/default/jenkins-infra.yaml

```diff
@@ -297,235 +297,95 @@ jenkins:
         jobs-settings: |
           jobs:
             - script: >
```

Comment:
Would it be possible to have a comment that explain the intend of this script and comment the other script as well

Reply:
The idea is to make it clearier where to add a git repository

Reply:
Good point!

Reply:
Done in #1126